docs/2.0/docs/accountfactory/installation/index.md

```diff
@@ -2,23 +2,20 @@
 
 ## Overview
 
-Account Factory is automatically integrated into [new Pipelines root repositories](/2.0/docs/pipelines/installation/addingnewrepo) during the bootstrapping process.
+Account Factory is automatically integrated into [new Pipelines root repositories](/2.0/docs/accountfactory/installation/addingnewrepo) during the bootstrapping process.
 
 By default, Account Factory includes the following components:
 
 - 📋 An HTML form for generating workflow inputs: `.github/workflows/account-factory-inputs.html`
```

Comment:
nit: I think these icons are a bit of an AI-callsign, and is not a pattern mirrored elsewhere in our docs.  Whilst I like a good emoji, I'm not sure these help

Reply:
Also, I know you didn't add these! Just a whilst-your-in-there thing